src/modules/message/store/action.js

```diff
@@ -3,11 +3,19 @@ import { cryptography } from '@liskhq/lisk-client';
 import { signMessageUsingHW } from '@wallet/utils/signMessage';
 import { signMessageWithPrivateKey } from '../utils/signMessageWithPrivateKey';
 
+export const getUnsignedMessage = (message) =>
```

Comment:
I think we can go with 
```suggestion
export const getNonProtocolMessage = (message) =>
```